src/ast/ast.h

```diff
@@ -212,6 +212,20 @@ bool AST_ClauseContainsAggregation
 	const cypher_astnode_t *clause
 );
 
+// returns the alias of a projection
+const char *AST_GetProjectionAlias
+(
+	const cypher_astnode_t* projection
+);
+
+// returns the alias of a projection
+// the alias will be NULL in the case of an unaliased non-identifier projection
+// returned in a Call {} clause
+const char *AST_GetSubqueryProjectionAlias
+(
+	const cypher_astnode_t* projection
+);
+
```

Comment:
Why can't we just use `AST_GetProjectionAlias` for both cases?
A `const cypher_astnode_t* projection` is a `const cypher_astnode_t* projection`

Reply:
That is what we do , the function declaration was not removed